### DISCUSSION TITLE
How to change the highlights for the text and buttons from the confirm?

### DISCUSSION THREAD

**Discussion by augustocdias** [folke/noice.nvim#838] [Q&A] [answered]
I've managed to change the border and the window color. How can I change the highlight for the buttons, title and the question text?

Another question. Is there a keybinding for focusing on another answer (just focusing, not "clicking")? 

![image](https://github.com/folke/noice.nvim/assets/1080789/c875ce97-49b6-4187-a882-86adffd0d925)

**Comment by ilias777** [comment, marked as answer]
You can change the highlights:

* `NoiceFormatConfirm`
* `NoiceFormatConfirmDefault`
